spec/controllers/physical_rack_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe PhysicalRackController do
+  render_views
+
+  let!(:server) { EvmSpecHelper.local_miq_server(:zone => zone) }
+  let(:zone) { FactoryGirl.build(:zone) }
```

Comment:
If you're not using the `server` as a variable at all, you can just put it into the `before` block below and not bind it.
```ruby
before do
  EvmSpecHelper.local_miq_server(:zone => FactoryGirl.build(:zone))
  ...
end
```